docs/content/getting-started/setup-openfga.mdx

```diff
@@ -119,10 +122,7 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=password
     healthcheck:
-      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
-      interval: 5s
-      timeout: 5s
-      retries: 5
+        test: ["CMD", 'mysqladmin', 'ping', '-h', 'localhost', '-u', 'root', '-p$$MYSQL_ROOT_PASSWORD' ]
```

Comment:
If this is a postgres container, why did we switch the healthcheck to mysql?

Reply:
oops